auth-server/src/services/auth.ts

```diff
@@ -273,6 +276,35 @@ export default class AuthService {
 			throw new ForbiddenError(messages.USER_EMAIL_ALREADY_EXISTS);
 		}
 
+		let user = await getUserFromUserId(userId);
+
+		const existingUndoToken = await UndoEmailChangeToken
+			.query()
+			.where('user_id', userId)
+			.orderBy('id', 'desc')
```

Comment:
Shouldn't we rather order by `created_at` to make sure we get the last one, independently from the id? I know it's most probably the same, but it feels more natural.

Reply:
done